packages/block-library/src/calendar/index.php

```diff
@@ -40,11 +40,17 @@ function render_block_core_calendar( $attributes ) {
 		}
 	}
 
+	$inline_styles = styles_for_block_core_calendar( $attributes );
+	$color_classes = get_color_classes_for_block_core_calendar( $attributes );
```

Comment:
We should probably leverage the style engine to generate all the color class names and inline styles. See [`lib/block-supports/colors.php`](https://github.com/WordPress/gutenberg/blob/trunk/lib/block-supports/colors.php) for how the block supports do this when serialization isn't skipped.